integration/workflow_registry_test.go

```diff
@@ -129,7 +129,7 @@ func TestRegistryFlow_JSON(t *testing.T) {
 		Sources: map[string]workflow.Source{
 			"test-source": {
 				Inputs: []workflow.Document{
-					{Location: "https://raw.githubusercontent.com/OAI/OpenAPI-Specification/main/examples/v3.0/petstore.json"},
+					{Location: "https://raw.githubusercontent.com/OAI/OpenAPI-Specification/main/examples/v3.0/petstore.yaml"},
```

Comment:
This test is called `TestRegistryFlow_JSON` for a reason 😅

Reply:
oops! Bit too big a find/replace